web/viewer.css

```diff
@@ -627,8 +627,8 @@ body {
   margin: 5px;
 }
 
-#findMsg {
-  color: rgba(251, 0, 0, 1);
+#findMsg[data-status="notFound"] {
```

Comment:
This should probably also be done for the "reached bottom, continuing from top" text that was previously also displayed in red, since it now feels a bit inconsistent that both are displayed in white now but only the "not found" text is bold.

Reply:
In the Firefox browser-findbar *only* the "notFound" message is displayed in bold, and I simply copied that behaviour here; hence I'm inclined to leave this as-is here.